socialwiki documentation/server update.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/expect -f
```

Comment:
please rename the file without a space, and also maybe set the username to be an argument
(note: committing these changes to your github will automatically update the pull request)